README.md

```diff
@@ -1,11 +1,33 @@
 # Bitcoinex
 
-Bitcoinex is striving to be the best and up-to-date Bitcoin Library for Elixir Developer.
+Bitcoinex is striving to be the best and up-to-date Bitcoin Library for Elixir.
```

Comment:
Sentence isn't grammatically correct: 

Bitcoinex is striving to be the best and *most* up-to-date Bitcoin Library for Elixir